### PR TITLE
Meteor-Auth0-Sample

### PR DESCRIPTION
**Pull request by jose-e-rodriguez**

### CODE REVIEW

File: settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "Auth0Domain": "yourAuth0domain",
```
Comment (ntotten) [inline settings.json]: `yourAuth0domain` should be `{DOMAIN}`

File: settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "Auth0ClientID": "yourAuth0clientID",
```
Comment (ntotten) [inline settings.json]: `yourAuth0clientID` should be `{CLIENT_ID}`

File: settings.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  "Auth0ClientSecret": "yourAuth0clientSecret",
```
Comment (ntotten) [inline settings.json]: `yourAuth0clientSecret` should be `{CLIENT_SECRET}`

**Comment by ntotten** [comment]
@jose-e-rodriguez Rather than put this in the root of the project lets do this: 

1. get rid of all the files in the root except for README.md
2. Update the readme to be like this: https://github.com/auth0/auth0-python
3. Add the LICENSE file: https://github.com/auth0/auth0-python/blob/master/LICENSE
4. Move the sample into the `examples/some-name-for-sample` folder (pick an appropriate name for the sample)

**Comment by jose-e-rodriguez** [comment]
Hello @ntotten 
The required changes were made. Please verify and let me know if it needs any other adjustments, I'll be pleased to update it as soon as possible.

**Comment by mirstan** [comment]
wow, you completely blew away base meteor auth0 package. why are you doing this?

**Comment by ntotten** [comment]
@mirstan It shouldnt be needed anymore. The new sample simplifies things greatly with new versions of meteor.

**Comment by ntotten** [comment]
@mirstan On second thought, we chatted about this and will bring back the old libraries.

**Comment by mirstan** [comment]
@ntotten any updates on restoring the module? can you also release the new version once it is done?

**Pull request merged**